client/java/src/main/java/com/linecorp/centraldogma/client/Watcher.java

```diff
@@ -32,11 +33,14 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.MissingNode;
 
+import com.linecorp.centraldogma.common.EntryNotFoundException;
+import com.linecorp.centraldogma.common.PathPattern;
 import com.linecorp.centraldogma.common.Query;
 import com.linecorp.centraldogma.common.Revision;
 
 /**
- * Watches the changes of a repository or a file.
+ * Watches the changes of a repository or a file. The {@link Watcher} must be closed via
+ * {@link Watcher#close()} after use.
```

Comment:
How about using `com.google.errorprone.annotations.MustBeClosed`?
I don't think we need to handle it in this PR. Let me create an issue.